# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/libwebrtc/call/audio_receive_stream.h
# Commit: d0b311007c03
# Full Hash: d0b311007c033e83824f5f6996a70ab9e870f31f
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Get RTCP BYE and RTP timeout handling working again (from Bug 1595479) r=mjf,dminor
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106145
# ==============================================================================

diff -r daa9eac709dc -r d0b311007c03 third_party/libwebrtc/call/audio_receive_stream.h
--- a/third_party/libwebrtc/call/audio_receive_stream.h	Fri Feb 19 16:20:38 2021 +0000
+++ b/third_party/libwebrtc/call/audio_receive_stream.h	Fri Feb 19 15:56:41 2021 -0600
@@ -19,6 +19,7 @@
 #include "absl/types/optional.h"
 #include "api/audio_codecs/audio_decoder_factory.h"
 #include "api/call/transport.h"
+#include "modules/rtp_rtcp/include/rtp_rtcp_defines.h"
 #include "api/crypto/crypto_options.h"
 #include "api/crypto/frame_decryptor_interface.h"
 #include "api/frame_transformer_interface.h"
@@ -125,6 +126,8 @@
 
       // RTP header extensions used for the received stream.
       std::vector<RtpExtension> extensions;
+
+      RtcpEventObserver* rtcp_event_observer = nullptr;
     } rtp;
 
     Transport* rtcp_send_transport = nullptr;